docs/user_guide/quickstart.rst

```diff
@@ -1,158 +1,135 @@
+
 Quickstart guide
-===================================
+================
 
-In this quickstart guide, we will illustrate core concepts of this chip implementation build flow by
-translating a simple Verilog based design into a GDSII IC layout database using
-the :ref:`freepdk45` virtual PDK.
+After following the :ref:`installation` instructions, you can either `run remotely`_ in the cloud, or `run locally`_ on your machine. The run instructions below will use a simple :ref:`asic_demo` using the :ref:`skywater130` PDK (set up from  :ref:`installation`).
```

Comment:
Maybe some more descriptive text for the `asic_demo` ref? "self-test target", "demo target", "demo ASIC design", something like that?